src/app/components/hris/employees/employee-approvals/employee-approvals.component.ts

```diff
@@ -43,6 +43,10 @@ export class EmployeeApprovalsComponent {
   screenWidth: number = 992;
   dataSource: MatTableDataSource<any> = new MatTableDataSource();
   dialogTypeData!: Dialog;
+  APPROVED = 0;
```

Comment:
Rather make these into enums in a separate file

Reply:
Created Enum file